### PR TITLE
Martinh/ntt cli commands

### PR DESCRIPTION
**Pull request by martin0995**
### Description

A page for NTT CLI commands.

### Checklist

- [x] **Required** - I have added a label to this PR 🏷️
- [x] **Required** - I have run my changes through Grammarly
- [ ] If pages have been moved, I have created redirects in the `wormhole-mkdocs` repo

### CODE REVIEW

**Review by PaulineAnnBar — COMMENTED** [review]
This is awesome! Can you please double check in the latest CLI upgrade if they didn't add isStandardRelayerSetTrue (can't remember the exact name!haha) but nikhil told me the PR was in review

File: build/contract-integrations/native-token-transfers/cli-commands.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: NTT CLI Commands
+description: A comprehensive guide to the NTT CLI detailing commands for managing native token transfers within Wormhole.
```
Comment (ilariae) [inline build/contract-integrations/native-token-transfers/cli-commands.md:3]: The description needs to be at least 120 characters

Comment (eshaben) [inline build/contract-integrations/native-token-transfers/cli-commands.md:3]:
```suggestion
description: A comprehensive guide to the Native Token Transfers (NTT) CLI, detailing commands for managing token transfers across chains within the Wormhole ecosystem.
```

**Review by ilariae — APPROVED** [review]
lgtm!

**Pull request merged**